akka-stream/src/main/scala/akka/stream/impl/io/IOSinks.scala

```diff
@@ -19,7 +19,7 @@ import scala.concurrent.{ Future, Promise }
  * Creates simple synchronous (Java 6 compatible) Sink which writes all incoming elements to the given file
```

Comment:
That comment should be removed in any case – Akka 2.4 itself is not 6 compatible, so it feels misleading to claim a specific class is